wordpress-mobile/WordPress-Android
# Issue/4539 reader related post track

Fixes #4539 - we now track when a related post is clicked in the reader detail.

To test:
* Locate a wp.com post in the reader list view
* Tap it to open the reader detail view
* Scroll to the bottom to show the "related reading" section
* Tap a related post
* Verify that `READER_RELATED_POST_CLICKED` is being tracked (it's done [here](https://github.com/wordpress-mobile/WordPress-Android/blob/5a86fe26047500ed912ac3fc0a7238f8996c9055/WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java#L543))

![screenshot_1474921865](https://cloud.githubusercontent.com/assets/3903757/18850729/c88217d0-8406-11e6-838e-c9438e7f5796.png)
